City-of-Helsinki/drupal-helfi-form-tool
# LOM-436: Fix accidently pushed dev asset versions

# [LOM-436](https://helsinkisolutionoffice.atlassian.net/browse/LOM-436)


## What was done


* Build assests without dev tags 

## How to install

* Make sure your instance is up and running on correct branch.
  * `git checkout LOM-436-fix-assets`
  * `make fresh`
* Run `make drush-cr`

## How to test


* [ ] If you want to test broken version checkout develop, add
* [ ]` $config['system.performance']['js']['preprocess'] = 1;` to local.settings.php
* Checkout to branch and `make drush-cr`, check that form works correctly




[LOM-436]: https://helsinkisolutionoffice.atlassian.net/browse/LOM-436?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
